### PR TITLE
Fix #2402: Resolve ktlint max line in home test package

### LINKED ISSUE #2402: Resolve ktlint max line in home test package
**Describe the bug**
We are preventing the 100 char length at some places like test case fun name. 
`// ktlint-disable max-line-length`

**Expected behavior**
We need to remove this prevention and fix the function name under 100 char length.

### PR DESCRIPTION
**Pull request by lakshaygarg128**
Fixes #2402 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by lakshaygarg128** [comment]
@anandwana001 pl. check it

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt [hunk 1]
```diff
@@ -179,7 +179,7 @@ class HomeActivityTest {
   }
 
   @Test
-  fun testHomeActivity_recyclerViewIndex0_displayGreetingMessageBasedOnTime_goodMorningMessageDisplayedSuccessful() { // ktlint-disable max-line-length
+  fun testHomeActivity_displayGreetMessageBasedOnTime_goodMorningMessageDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]:
```kotlin
fun testHomeActivity_morningTimestamp_goodMorningMessageIsDisplayed()
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks, @lakshaygarg128 
Left a comment, apply the same for others too. 

```diff
- DisplayedSuccessfully
+ IsDisplayed

- testHomeActivity_recyclerViewIndex1_displaysRecentlyPlayedStoriesText
+ testHomeActivity_recentlyPlayedStoriesHeadingIsDisplayed

- testHomeActivity_recyclerViewIndex1_displaysViewAllText
+ testHomeActivity_viewAllHeadingIsDisplayed

- testHomeActivity_recyclerViewIndex1_clickViewAll_opensRecentlyPlayedActivity
+ testHomeActivity_clickViewAll_opensRecentlyPlayedActivity
```

remove all `recyclerViewIndex` things from the function name

**Comment by lakshaygarg128** [comment]
> Thanks, @lakshaygarg128
> Left a comment, apply the same for others too.
> 
> ```diff
> - DisplayedSuccessfully
> + IsDisplayed
> 
> - testHomeActivity_recyclerViewIndex1_displaysRecentlyPlayedStoriesText
> + testHomeActivity_recentlyPlayedStoriesHeadingIsDisplayed
> 
> - testHomeActivity_recyclerViewIndex1_displaysViewAllText
> + testHomeActivity_viewAllHeadingIsDisplayed
> 
> - testHomeActivity_recyclerViewIndex1_clickViewAll_opensRecentlyPlayedActivity
> + testHomeActivity_clickViewAll_opensRecentlyPlayedActivity
> ```
> 
> remove all `recyclerViewIndex` things from the function name

the function without ``// ktlint-disable max-line-length`` with ``recyclerViewIndex`` should also be removed ??
@anandwana001

**Comment by anandwana001** [comment]
Yes, you can fix it, it will be very helpful.

**Comment by lakshaygarg128** [comment]
@anandwana001 pl. review it .

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt (hunk 1 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt [hunk 2]
```diff
@@ -236,7 +236,7 @@ class HomeActivityTest {
   }
 
   @Test
-  fun testHomeActivity_recyclerViewIndex1_displaysRecentlyPlayedStoriesText() {
+  fun testHomeActivity_displaysRecentlyPlayedStoriesText() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: testHomeActivity_recentlyPlayedStoriesTextIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt [hunk 3]
```diff
@@ -257,7 +257,7 @@ class HomeActivityTest {
   }
 
   @Test
-  fun testHomeActivity_recyclerViewIndex1_displaysViewAllText() {
+  fun testHomeActivity_displaysViewAllText() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: testHomeActivity_viewAllTextIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt [hunk 4]
```diff
@@ -441,7 +441,7 @@ class HomeActivityTest {
   }
 
   @Test
-  fun testHomeActivity_recyclerViewIndex4_topicSummary_configurationChange_lessonCountIsCorrect() {
+  fun testHomeActivity_topicSummary_configurationChange_lessonCountIsCorrect() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: let's use configChange to bring consistency

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 5]
```diff
@@ -177,7 +177,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_recyclerViewItem0_doesNotShowSectionDivider() {
+  fun testRecentlyPlayedTestActivity_doesNotShowSectionDivider() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_sectionDividerIsNotDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 6]
```diff
@@ -198,7 +198,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_recyclerViewItem0_showsLastWeekSectionTitle() {
+  fun testRecentlyPlayedTestActivity_showsLastWeekSectionTitle() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_lastWeekSectionTitleIsDIsplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 7]
```diff
@@ -277,7 +277,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_recyclerViewItem1_clickStory_intentsToExplorationActivity() {
+  fun testRecentlyPlayedTestActivity_clickStory_intentsToExplorationActivity() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_clickStory_opensExplorationActivity

**Review by anandwana001 — CHANGES_REQUESTED** [review]
@lakshaygarg128 PTAL

### HomeActivityTest.kt
```diff
- testHomeActivity_recyclerViewIndex4_topicSummary_topicNameIsCorrect
+ testHomeActivity_secondTestTopic_topicSummary_topicNameIsCorrect

// I missed this one earlier 
- testHomeActivity_topicSummary_topicNameIsCorrect
+ testHomeActivity_firstTestTopic_topicSummary_topicNameIsCorrect

- testHomeActivity_topicSummary_lessonCountIsCorrect
+ testHomeActivity_fiveLessons_topicSummary_lessonCountIsCorrect

- testHomeActivity_recyclerViewIndex4_topicSummary_lessonCountIsCorrect
+ testHomeActivity_oneLesson_topicSummary_lessonCountIsCorrect

- testHomeActivity_topicSummary_configurationChange_lessonCountIsCorrect
+ testHomeActivity_oneLesson_topicSummary_configChange_lessonCountIsCorrect

- testHomeActivity_onBackPressed_showsExitToProfileChooserDialog
+ testHomeActivity_onBackPressed_exitToProfileChooserDialogIsDisplayed

- testHomeActivity_onBackPressed_orientationChange_showsExitToProfileChooserDialog
+ testHomeActivity_onBackPressed_configChange_exitToProfileChooserDialogIsDisplayed

- testHomeActivity_onBackPressed_clickExit_checkOpensProfileActivity
+ testHomeActivity_onBackPressed_clickExit_opensProfileActivity
```

### RecentlyPlayedFragmentTest.kt
```diff
- testRecentlyPlayedTestActivity_toolbarTitle_isDisplayedSuccessfully
+ testRecentlyPlayedTestActivity_toolbarTitle_isDisplayed

- testRecentlyPlayedTestActivity_recyclerViewItem2_showsLastMonthSectionTitle
+ testRecentlyPlayedTestActivity_lastMonthSectionTitleIsDIsplayed

- testRecentlyPlayedTestActivity_recyclerViewItem2_showsSectionDivider
+ testRecentlyPlayedTestActivity_sectionDividerIsDisplayed

- testRecentlyPlayedTestActivity_changeConfiguration_toolbarTitle_isDisplayedSuccessfully
+ testRecentlyPlayedTestActivity_configChange_toolbarTitle_isDisplayed

- testRecentlyPlayedTestActivity_configurationChange_checkSpanForItem1_spanSizeIsOne
+ testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne

- testRecentlyPlayedTestActivity_configurationChange_checkSpanForItem3_spanSizeIsOne
+ testRecentlyPlayedTestActivity_configChange_checkSpanForItem3_spanSizeIsOne
```

**Comment by lakshaygarg128** [comment]
Done all the changes requested by you . @anandwana001

**Comment by Luffy18346** [comment]
@lakshaygarg128 If all changes mentioned by @anandwana001 is done from your side, then please reply to all the comments made by @anandwana001.
For eg, "Changes done" or  simply "Done".

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt (hunk 2 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt (hunk 3 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt (hunk 4 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 5 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 6 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 7 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 8]
```diff
@@ -377,7 +377,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_changeConfiguration_recyclerViewItem0_doesNotShowSectionDivider() { // ktlint-disable max-line-length
+  fun testRecentlyPlayedTestActivity_changeConfig_doesNotShowSectionDivider() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: configChange

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 9]
```diff
@@ -397,7 +397,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_changeConfiguration_recyclerViewItem0_showsLastWeekSectionTitle() { // ktlint-disable max-line-length
+  fun testRecentlyPlayedTestActivity_changeConfig_showsLastWeekSectionTitle() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: configChange

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 10]
```diff
@@ -463,7 +463,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_changeConfiguration_recyclerViewItem1_lessonThumbnailIsCorrect() { // ktlint-disable max-line-length
+  fun testRecentlyPlayedTestActivity_changeConfig_lessonThumbnailIsCorrect() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: configChange

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 11]
```diff
@@ -485,7 +485,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_changeConfiguration_recyclerViewItem2_showsLastMonthSectionTitle() { // ktlint-disable max-line-length
+  fun testRecentlyPlayedTestActivity_changeConfig_showsLastMonthSectionTitle() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: configChange

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 8 above)
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_configChange_sectionDividerIsNotDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 9 above)
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_configChange_lastWeekSectionTitleIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 11 above)
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_configChange_lastMonthSectionTitleIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 12]
```diff
@@ -528,7 +528,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_checkSpanForItem3_spanSizeIsOne() {
+  fun testRecentlyPlayedTestActivity_configChange_checkSpanForItem3_spanSizeIsOne() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: For test at line 552
```diff
- testRecentlyPlayedTestActivity_configurationChange_checkSpanForItem1_spanSizeIsOne
+ testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne
```

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: for others too, `configChange`. Please keep the consistency in all the names.

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 8 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 9 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 10 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 11 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 8 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 9 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 11 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 12 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: There are two test with same name 
``testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne``
 but one is for landscape and other is of portrait
@anandwana001

Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: So we have to modify the name according to that

Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: Should i change it to
``testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne`` to 
``testRecentlyPlayedTestActivity_configChangeLand_checkSpanForItem1_spanSizeIsOne``

and  ``configChangePot`` for portrait

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: the test with name `testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne` is wrong as there is no configuration change happening in the test. 

update to 
testRecentlyPlayedTestActivity_checkSpanForItem1_spanSizeIsOne

Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: ok

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 13]
```diff
@@ -198,7 +198,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_recyclerViewItem0_showsLastWeekSectionTitle() {
+  fun testRecentlyPlayedTestActivity_lastWeekSectionTitleIsDIsplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_lastWeekSectionTitleIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 12 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 14]
```diff
@@ -317,7 +317,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_recyclerViewItem2_showsLastMonthSectionTitle() {
+  fun testRecentlyPlayedTestActivity_lastMonthSectionTitleIsDIsplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_lastMonthSectionTitleIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 15]
```diff
@@ -361,7 +361,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_changeConfiguration_toolbarTitle_isDisplayedSuccessfully() {
+  fun testRecentlyPlayedTestActivity_configChange_toolbarTitle_isDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_configChange_toolbarTitleIsDisplayed

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt [hunk 16]
```diff
@@ -159,7 +159,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_toolbarTitle_isDisplayedSuccessfully() {
+  fun testRecentlyPlayedTestActivity_toolbarTitle_isDisplayed() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: testRecentlyPlayedTestActivity_toolbarTitleIsDisplayed

**Review by anandwana001 — CHANGES_REQUESTED** [review]
check for `IsDIsplayed` -> `IsDisplayed` 
`toolbarTitle_isDisplayed` -> `toolbarTitleIsDisplayed`

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 13 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: DONE

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 14 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: already done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 15 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

File: app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt (hunk 16 above)
Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: done

Comment (lakshaygarg128) [reply app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt]: @anandwana001 PTAL

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Resolve the conflict and we are good to merge. 
Thanks, @lakshaygarg128

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/main/res/layout-land/profile_progress_header.xml [hunk 17]
```diff
@@ -51,6 +51,8 @@
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="28sp"
+      android:textAlignment="viewStart"
+      app:layout_constraintStart_toEndOf="@+id/profile_edit_image"
```
Comment (anandwana001) [inline app/src/main/res/layout-land/profile_progress_header.xml]: Why these changes?

**Review by anandwana001 — CHANGES_REQUESTED** [review]
@lakshaygarg128 PTAL

File: app/src/main/res/layout-land/profile_progress_header.xml (hunk 17 above)
Comment (lakshaygarg128) [reply app/src/main/res/layout-land/profile_progress_header.xml]: Sorry I think the previous issue code changes might be pushed ... now it is removed @anandwana001

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @lakshaygarg128

**Pull request merged**